tests/test_main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+from app.main import is_isogram
```

Comment:
You have tested only cases when the function returns `True`.
So the following function will pass all your tests:
```
is_isogram(word: str) -> bool:
    return True
```